### PR TITLE
feat: link to token page for AEX-X classified tokens

### LINKED ISSUE #435: link to token page for AEX-X classified tokens
### Please describe the problem that your request should resolve.
can't easily access the token page for an AEX-9 token from contract detail view
image

### Describe the solution you'd like
please provide a way to access the token page from the contract detail view in case it is classified as AEX-9 / AEX-141 (once the page exists)

### Is it already possible to achieve the same outcome in any other way? If so, how?
not really, as token search in token view is missing for unlisted tokens. more experienced users can copy the contractId and past it into the `/tokens/` url 😄

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #435

## Demo

https://github.com/aeternity/aescan/assets/15363559/59084af4-fca3-4346-9d1a-31d996e0b869

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add token link to smart contract detail

### CODE REVIEW

File: src/components/ContractDetailsPanel.vue [hunk 1]
```diff
@@ -12,12 +12,40 @@
           :label="formatEllipseHash(contractDetails.id)"
           :clipboard-text="contractDetails.id"/>
       </div>
-      <app-chip v-if="contractDetails?.contractType">
+      <app-chip
+        v-if="contractDetails?.contractType"
+        size="sm">
```
Comment (janmichek) [inline src/components/ContractDetailsPanel.vue:17]: unrelated fix

File: src/pages/nfts/[id].vue [hunk 2]
```diff
@@ -47,16 +47,18 @@ const hasTemplates = computed(() => {
...
-
-if (error.value) {
-  throw showError({
-    data: {
-      entityId: route.params.id,
-      entityName: 'NFT',
-    },
-    statusMessage: 'EntityDetailsNotFound',
-  })
+try {
```
Comment (janmichek) [inline src/pages/nfts/[id].vue:50]: Unrelated fix. I found that direct URL access to NFT detail page will cause infinite loading.
EG: https://aescan.dev.aepps.com/nfts/ct_2iJ5RHF2MbriceSetNiKARVii76Fr8Fi5wGY6kUHUj3wq7f3io
Could you help me to verify the solution @lukeromanowicz ?

File: src/components/ContractDetailsPanel.vue [hunk 3]
```diff
@@ -180,6 +209,10 @@ const props = defineProps({
   },
 })
 
+onMounted(() => {
+  console.log(props.contractDetails)
+})
+
```
Comment (michele-franchi) [inline src/components/ContractDetailsPanel.vue:215]:
```suggestion
```

**Review by michele-franchi — CHANGES_REQUESTED** [review]
I see some visual differences compared to other rows:
image

- There's a bigger vertical padding
- There's a right margin after the token name

Would it be possible to achieve the same row height of the other rows? Looks like we already to that for API Links?

File: src/components/ContractDetailsPanel.vue (hunk 3 above)
Comment (janmichek) [reply src/components/ContractDetailsPanel.vue]: oops forgot that. Fixed

**Comment by janmichek** [comment]
> I see some visual differences compared to other rows: image
> 
>     * There's a bigger vertical padding
> 
>     * There's a right margin after the token name
> 
> 
> Would it be possible to achieve the same row height of the other rows? Looks like we already to that for API Links?

Thanks for second eyes. Fixed

**Review by michele-franchi — APPROVED** [review]
Looks good, there's another one small issue but I approve in advance.

The entire td is clickable:
image
but ideally only the symbol and text part should be, could you adjust it?

**Comment by janmichek** [comment]
Thanks fixed

> Looks good, there's another one small issue but I approve in advance.
> 
> The entire td is clickable (because of flex): image but ideally only the symbol and text part should be, could you adjust it?

**Review by Liubov-crypto — APPROVED** [review]
LGTM

**Review by lukeromanowicz — APPROVED** [review]
Looks good :+1:

File: src/pages/nfts/[id].vue (hunk 2 above)
Comment (janmichek) [reply src/pages/nfts/[id].vue:50]: Thanks, I created the followup issue. Can you provide some hint how to improve it there? https://github.com/aeternity/aescan/issues/517 I am bit out of ideas. The 'correct' approach does not work when accessed directly. This solution it's not that effective but at least works

**Pull request merged**